src/internal/m365/collection/site/collection.go

```diff
@@ -81,6 +81,10 @@ func NewCollection(
 	return c
 }
 
+func (sc *Collection) SetBetaService(betaService *betaAPI.BetaService) {
```

Comment:
Added an additional function to help with setting beta service as the field is private.